### PR TITLE
Add resume test

### PR DESCRIPTION
**Pull request by JoshOldenburg**

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -51,15 +52,19 @@ parameters:
     - '#Parameter \#1 \$data of function bin2hex expects string, string\|false given\.#'
     - '#Parameter \#1 \$query of method Illuminate\\Database\\Eloquent\\Builder<App\\Models\\Attendance>::selectSub\(\) expects Closure\|Illuminate\\Database\\Eloquent\\Builder<App\\Models\\Attendance>\|string, Illuminate\\Database\\Eloquent\\Builder<App\\Models\\User> given\.#'
     - '#Parameter \#1 \$str of function strtolower expects string, string\|null given\.#'
+    - '#Parameter \#1 \$string of function strlen expects string, string\|false given\.#'
+    - '#Parameter \#1 \$user of method Illuminate\\Foundation\\Testing\\TestCase::actingAs\(\) expects Illuminate\\Contracts\\Auth\\Authenticatable, mixed given\.#'
```
Comment (kberzinch) [inline phpstan.neon:56]: Should we just ignore the `tests` directory entirely?

File: tests/Feature/ResumeTest.php [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        });
+        Storage::disk('local')->assertMissing('resumes/'.$user->uid.'.pdf');
+
+        // Check an upload of a TXT fails
+        $response = $this->actingAs($user, 'web')->post('/api/v1/users/'.$user->id.'/resume', [
+            'resume' => $textFile,
+        ]);
+        $response->assertStatus(400);
+        $response->assertJson(static function (AssertableJson $json): void {
+            // This should be resume_not_pdf, but exiftool does not handle txt files nicely.
```
Comment (kberzinch) [inline tests/Feature/ResumeTest.php:125]: Perhaps we could use the `file` command to guess the type before calling `exiftool`.

**Review by kberzinch — APPROVED** [review]
Looks good. A few discussion topics...

File: tests/Feature/ResumeTest.php (hunk 2 above)
Comment (JoshOldenburg) [reply tests/Feature/ResumeTest.php:125]: Good point. I'll open an issue.

File: phpstan.neon (hunk 1 above)
Comment (JoshOldenburg) [reply phpstan.neon:56]: If it keeps requiring ignores then sure.

**Pull request merged**